### ISSUE TITLE
hawtapp tarballs don't have jolokia enabled by default

### ISSUE THREAD

**Issue by jstrachan** [fabric8io/fabric8#5720]
we should enable jolokia / agentbond by default on all hawtapp tarballs when run via the `bin/run.sh` command then we can get the hawtio console working with fabric8 ansible

**Comment by davsclaus** [comment]
Yep as @chirino is working on that plugin maybe he can take a look?
